### ISSUE TITLE
JSONDecodeError: Expecting value: line 1 column 1

### ISSUE THREAD

**Issue by GitCat12** [nathom/streamrip#122]
## Bug Report

**Current Behavior**
```
return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Command used:**
rip search -s soundcloud 'Revolution by the score'

```bash
rip search -s soundcloud 'Revolution by the score'
```

**Expected behavior**
Bring up the download menu

**Comment by nathom** [comment]
Fixed in 0.7.1

**Comment by GitCat12** [comment]
> Fixed in 0.7.1

Thanks!

**Comment by ryansalerno** [comment]
Try also `rip url https://play.qobuz.com/album/5051083020763`. The formatting of the error looks different, but the message is the same (I'm assuming due to being on a much newer version).
